crates/sys/src/lib.rs

```diff
@@ -22,8 +22,9 @@ mod inlines;
     non_upper_case_globals,
     non_camel_case_types,
     non_snake_case,
-    clippy::transmute_int_to_bool,
     clippy::useless_transmute,
+    clippy::transmute_int_to_bool,
+    clippy::missing_safety_doc,
```

Comment:
Please minimise the diff here:
```suggestion
    clippy::missing_safety_doc,
    clippy::transmute_int_to_bool,
    clippy::useless_transmute,
```